pkgs/applications/version-management/radicle-cli/default.nix

```diff
@@ -2,86 +2,38 @@
 , stdenv
 , fetchFromGitHub
 , rustPlatform
-, pkg-config
-, cmake
-, installShellFiles
-, asciidoctor
 , DarwinTools
-, openssl
-, libusb1
 , AppKit
-, git
-, openssh
-, testers
-, radicle-cli
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "radicle-cli";
-  version = "0.6.1";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "radicle-dev";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-LS6zYpMg0LanRL2M8ioGG8Ys07TPT/3hP7geEGehwxg=";
+    repo = "heartwood";
+    rev = "b27313bc5c703c8a25343176c620c8652b815abc";
+    sha256 = "sha256-WyLqTOF1TOuRD4/aTikeOR446fd00opryiUnAdPfA9Q=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "automerge-0.0.2" = "sha256-MZ1/rca8ZsEUhd3bhd502PHlBbvqAOtnWFEdp7XWmYE=";
-      "automerge-0.1.0" = "sha256-dwbmx3W13oZ1O0Uw3/D5Z0ht1BO1PmVVoWc/tLCm0/4=";
-      "cob-0.1.0" = "sha256-ewPJEx7OSr8X6e5QJ4dh2SbzZ2TDa8G4zBR5euBbABo=";
-      "libusb1-sys-0.6.2" = "sha256-577ld1xqJkHp2bqALNq5IuZivD8y+VO8vNy9Y+hfq6c=";
-      "walletconnect-0.1.0" = "sha256-fdgdhotTYBmWbR4r0OMplOwhYq1C7jkuOdhKASjH+Fs=";
-    };
-  };
-
-  # Otherwise, there are errors due to the `abigen` macro from `ethers`.
-  auditable = false;
+  cargoHash = "sha256-ZKIGaHTIvlkVIPhuGOYTG0zXBOZoNxtcNIVZJyuNa+8=";
 
-  nativeBuildInputs = [
-    pkg-config
-    cmake
-    installShellFiles
-    asciidoctor
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+  nativeBuildInputs = [ ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
     DarwinTools
   ];
 
-  buildInputs = [
-    openssl
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    libusb1
+  buildInputs = [ ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
     AppKit
   ];
 
-  postInstall = ''
-    for f in $(find . -name '*.adoc'); do
-      mf=''${f%.*}
-      asciidoctor --doctype manpage --backend manpage $f -o $mf
-      installManPage $mf
-    done
-  '';
-
-  nativeCheckInputs = [
-    git
-    openssh
-  ];
-  preCheck = ''
-    eval $(ssh-agent)
-  '';
-
-  passthru.tests = {
-    version = testers.testVersion { package = radicle-cli; };
-  };
+  doCheck = false;
```

Comment:
Any specific reason for disabling this? I.e. most tests pass with `nativeCheckInputs = [ git ];`, but there are some failures that might require further investigation.

Reply:
Yeah we saw some issues, with running `cargo test` in the `installCheckPhase`.
Opened a topic in the Radicle chat to see if we can debug the errors that require further investigation.
https://radicle.zulipchat.com/#narrow/stream/369277-heartwood/topic/nix.20package/near/376875202

Reply:
I think you can eliminate the other failures by adding

```
cargoTestFlags = [
    "--all"
  ];
```

Reply:
PS. I think we should also add `cargoBuildFlags = ["--workspace"];` so we get `radicle-httpd` and `radicle-tui` too

Reply:
Progress: much more tests pass with
```nix
  cargoTestFlags = [ "--workspace" ];
  checkType = "debug";
  nativeCheckInputs = [ git tree ];
```
But there are still some failures left...

Reply:
Thanks at all for your time and the awesome suggestions. I've tried to combine all inputs into a fresh commit.
However, in case I activate the tests, I cannot get it to succeed.
Latest error message I get is something with `rad_sync`, see below. Might be, because I removed some dependencies?
```
       > failures:
       >     rad_sync
       >
       > test result: FAILED. 33 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in
```

Reply:
thread 'rad_sync' panicked at 'Node::routes_to: unexpected route for ...
... radicle-cli/tests/commands.rs:792:9

Reply:
Fixed by removing the race conditions (of the multiple threads):
`dontUseCargoParallelTests = true`